dev/ci/ci-mathcomp_word.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+git_download mathcomp_word
+
+if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
+
+( cd "${CI_BUILD_DIR}/mathcomp_word"
+  dune build @install -p coq-mathcomp-word
```

Comment:
It seems dune doesn't find the installed mathcomp (in user-contrib) leading to the failure in https://gitlab.com/coq/coq/-/jobs/2250030795
@ejgallego any tips?

AFAICT this is the first use of dune to compile 3rd party .v files in the CI (menhirlib uses dune but I think only for the ocaml part). Maybe we would be better off using coq_makefile?

Reply:
I can't even reproduce this problem locally. For me it builds fine, I get the problem only between mathcomp-word and Jasmin, not between mathcomp & mathcomp-word.

Reply:
Yes, I think what makes it work locally is that `COQLIB` is set to the right location: https://github.com/coq/coq/blob/master/dev/ci/ci-common.sh#L44

Whereas in the gitlab case (a few lines above), we don't seem to set `COQLIB`.

@SkySkimmer do you know why?

Reply:
I can't look at that today, you mind opening a bug so i can see what's going on next week?

Reply:
In the gitlab case coq should be configured and installed so it already knows where coqlib is.